frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -31,15 +31,20 @@ common_settings.network = {
     text = _("Network"),
     sub_item_table = NetworkMgr:getMenuTable()
 }
+
 common_settings.screen = {
     text = _("Screen"),
     sub_item_table = {
         require("ui/elements/screen_dpi_menu_table"),
         require("ui/elements/screen_eink_opt_menu_table"),
         require("ui/elements/screen_disable_double_tap_table"),
-        require("ui/elements/refresh_menu_table"),
+        require("ui/elements/refresh_menu_table")
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
It seems there's a problem in that line.
I believe it's _common_settings.screen.sub_item_table_ could you help me get reference to above _sub_item_table_?